docs/magento2/commerce.mdx

```diff
@@ -185,3 +212,180 @@ filters on ` option_id` instead of `value_id`.
 If it isn't the case, there is an issue with the patching process in your
 project. <ContactLink>Contact us and we'll help you to troubleshoot
 this</ContactLink>.
+
+```mdx-code-block
+  </div>
+</details>
+```
+
+<!--------------------DETAILS-------------------->
+
+```mdx-code-block
+<details>
+  <summary className="flex justify-between"><span>The <code>Yes/No</code> field returns the id instead of value to the end user</span><SinceVersion inline tag="2.22.1" /></summary>
```

Comment:
I find a bit odd to specify a FC version in a known issue so I would suggest to just remove it from here and leave the version in paragraph stating the field types support.

```suggestion
  <summary className="flex justify-between"><span>The <code>Yes/No</code> field returns the id instead of value to the end user</span></summary>
```